index.html

```diff
@@ -11620,6 +11620,40 @@ <h2>A project for first-time contributions</h2>
                 </div>
                 <!-- ________ Pulkit's card END ________  -->
 
+                <!--_________Harsh's card START ________-->
+                <div class="card">
+                    <p class="name">Harsh Agarwal/p>
```

Comment:
You need to close this _p_ tag using `</p>` at the end of the line

Reply:
No, just solve these in your branch, commit and push them, so the PR will be automatically updated.